README.md

```diff
@@ -20,7 +24,7 @@ Regionalized Value State Dependence Graph (RVSDG) as intermediate representation
 make all
 ```
 
-This presumes that llvm-config-17 can be found in $PATH. If that is not the case,
+This presumes that llvm-config-18 can be found in $PATH. If that is not the case,
```

Comment:
We had missed that 17 was used also for the llvm-config

Reply:
Maybe you can reformulate it such that we do not mention the version there. Suggestion: "This presumes that the right version of llvm-config can be found ..."

Reply:
Changed to suggestion